### PR TITLE
Suppress errors when submitting feedback

### PR DESCRIPTION
**Pull request by sammacbeth**
**Reviewer:**

## Description:
There is an issue with the feedback backend, meaning it reports an error, even though the feedback is successfully submitted. This fixes the client side by reporting that feedback was successfully submitted even if there was an error with the request.
https://app.asana.com/0/276630244458377/1200216229834078/f

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/ui/templates/feedback-form.es6.js [hunk 1]
```diff
@@ -16,7 +12,7 @@ module.exports = function () {
             <label class='frm__label'>Which website is broken?</label>
             <input class='js-feedback-url frm__input' type='text' placeholder='Copy and paste your URL' value='${this.model.url}'/>
             <label class='frm__label'>Describe the issue you encountered:</label>
-            <textarea class='frm__text js-feedback-message' placeholder='Which website content or functionality is broken? Please be as specific as possible.'></textarea>
+            <textarea class='frm__text js-feedback-message' required placeholder='Which website content or functionality is broken? Please be as specific as possible.'></textarea>
```
Comment (kdzwinel) [inline shared/js/ui/templates/feedback-form.es6.js:15]: Why this field is required and URL isn't?

Comment (sammacbeth) [reply shared/js/ui/templates/feedback-form.es6.js:15]: This form can also be used for standard feedback - without a broken URL.

Comment (kdzwinel) [reply shared/js/ui/templates/feedback-form.es6.js:15]: Makes sense 👍

**Review by jdorweiler — APPROVED** [review]

**Comment by jdorweiler** [comment]
👍

**Pull request merged**